rocket_component.py

```diff
@@ -0,0 +1,102 @@
+from atmosphere import Atmosphere
+import math
+
+
+class Rocket:
+  """
+  The rocket class will calculate the forces acting upon the rocket component
+  at a given time. Units will be metric for consistency.
+  """
+
+  def __init__(self) -> None:
+
+    # Basic Properties (Falcon 9 ex.)
+    self.altitude = 0  # m
```

Comment:
Should we just make this pos (position) instead and then we can use pos.y when we need the altitude?

Reply:
I'm fine with leaving it as altitude. it makes more sense as a variable name within the equations later on. like you said, when we go to call the function we can pass in pos.y

Reply:
Then why do we have a Position class?